hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/prometheus/PushGatewayMetricsReporter.java

```diff
@@ -77,4 +83,11 @@ private String getJobName() {
     }
     return configuredJobName;
   }
+
+  private static Map<String, String> parseLabels(String labels) {
+    return Pattern.compile("\\s*,\\s*")
+        .splitAsStream(labels.trim())
+        .map(s -> s.split(":", 2))
```

Comment:
What if labels are not in the format `key:value`? Should we add a validation or handle this scenario?